typings/objection/index.d.ts

```diff
@@ -391,6 +391,7 @@ declare namespace Objection {
   }
 
   interface WhereCompositeMethod<QB extends AnyQueryBuilder> {
+    (column: ColumnRef[], op: Operator, value: Value[]): QB;
```

Comment:
Pretty sure this was just missing, so I added it.

Reply:
Yep, thanks!